test/dlc_tests/conftest.py

```diff
@@ -248,6 +251,10 @@ def pytest_runtest_setup(item):
         canary_opts = [mark for mark in item.iter_markers(name="canary")]
         if not canary_opts:
             pytest.skip("Skipping non-canary tests")
+    if item.config.getoption("--eks-multi-node"):
+        eks_multinode_opts = [mark for mark in item.iter_markers(name="multinode")]
```

Comment:
let's generalize this as running multinode tests. as implemented, any test marked multinode would be run in this case -- there's nothing explicitly restricting to EKS here, so we should make sure the naming conventions reflect that

Reply:
can you remove "eks" from this variable name